### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by frangueriniwolox**

### CODE REVIEW

**Comment by frangueriniwolox** [comment]
## Summary

- Front-End Title component added
- Colors style-sheet created
- Fonts style-sheet created

## Screenshots

page-titles image

## Trello card

https://trello.com/c/tvbl6cFW/21-page-titles

File: index.html [hunk 1]
```diff
@@ -9,6 +9,6 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1> 
+  <h1 class="title">Front-End Titles</h1>
```
Comment (SKOLZ) [inline index.html]: the text should be `Front-End Tools` instead

File: scss/styles.scss [hunk 2]
```diff
@@ -1,3 +1 @@
-.title {
-  color: red;
-}
+
```
Comment (SKOLZ) [inline scss/styles.scss]: you can remove this file

File: scss/application.scss [hunk 3]
```diff
@@ -1,2 +1,5 @@
-@import 'scss/equalizer';
-@import 'scss/styles';
+@import 'equalizer';
+@import 'styles';
```
Comment (SKOLZ) [inline scss/application.scss]: remove this import

File: scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title:before{
+    background: url(assets/triangle.svg);
+    background-repeat: no-repeat;
+    content: '';
+    height: $title-decoration-height;
+    position: absolute;
+    right: calc(100% - 20px);
```
Comment (SKOLZ) [inline scss/titles.scss]: I thought we said we would use `left: 0;` here to keep the full padding approach

File: scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    width: $title-decoration-width;
+    z-index: -1;
+}
+
+.title {
+    color: $mine-shaft;
+    font-family: $font-stack;
+    font-size: $text-big;
+    line-height: 24px;
+    margin: 34px 0 34px 50px;
```
Comment (SKOLZ) [inline scss/titles.scss]: padding here

File: scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    position: absolute;
```
Comment (SKOLZ) [inline scss/titles.scss]: remember this should have position relative to be the origin of the absolute positioned child.

**Pull request merged**